qiskit_ibm_runtime/estimator.py

```diff
@@ -268,6 +251,15 @@ def run(  # pylint: disable=arguments-differ
             ValueError: Invalid arguments are given.
         """
         # To bypass base class merging of options.
+        if "noise_model" not in kwargs:
```

Comment:
`kwargs` here actually accepts both structured and flat dictionaries, e.g. 

```
run(noise_model=foo)
```

and 

```
run(simulator={"noise_model": foo})
```

So it's probably easier to do this logic in `_run()` after `_merge_options()`, which combines everything into a "structured" dictionary.

It's done this way b/c we want to support both auto-complete with `Options` (structured) and flat dictionary as defined by the base class, _and_ `asdict(options)`.